### PR TITLE
Doing a small review of documentation website pages and fixing the most eye catching failures

### PR DESCRIPTION
**Pull request by gretanausedaite**
## Changes

Added props table to ButtonGroup and Slider;
Updated jsdocs in InputGrid and LabeledInput components;
Fixed Tooltip in Avatar example;

## Testing

N/A

## Docs

N/A

### CODE REVIEW

File: packages/itwinui-react/src/core/LabeledInput/LabeledInput.tsx [hunk 1]
```diff
@@ -34,9 +34,6 @@ export type LabeledInputProps = {
   wrapperProps?: React.ComponentProps<typeof InputGrid>;
   /**
    * Set display style of label.
-   * Supported values:
-   * - 'default' - label appears above input.
-   * - 'inline' - appears in the same line as input.
```
Comment (rohan-kadkol) [inline packages/itwinui-react/src/core/LabeledInput/LabeledInput.tsx:39]: Why was this removed here and in `InputGrid.tsx`?

Comment (gretanausedaite) [reply packages/itwinui-react/src/core/LabeledInput/LabeledInput.tsx:39]: This style of comments is not used anywhere else with few options. Removed for consistency.

Comment (mayank99) [reply packages/itwinui-react/src/core/LabeledInput/LabeledInput.tsx:39]: but it's useful information

Comment (mayank99) [reply packages/itwinui-react/src/core/LabeledInput/LabeledInput.tsx:39]: also used in other places.

https://github.com/iTwin/iTwinUI/blob/5e46f431232a186968afa9c4b76d28604af64f5f/packages/itwinui-react/src/core/utils/components/Portal.tsx#L29-L33

https://github.com/iTwin/iTwinUI/blob/5e46f431232a186968afa9c4b76d28604af64f5f/packages/itwinui-react/src/core/utils/components/Icon.tsx#L13C2-L17

https://github.com/iTwin/iTwinUI/blob/5e46f431232a186968afa9c4b76d28604af64f5f/packages/itwinui-react/src/core/NotificationMarker/NotificationMarker.tsx#L16-L22

Comment (gretanausedaite) [reply packages/itwinui-react/src/core/LabeledInput/LabeledInput.tsx]: Added back. It has some css issues in website.

Comment (mayank99) [reply packages/itwinui-react/src/core/LabeledInput/LabeledInput.tsx]: shouldn't the css be fixed then?

**Review by mayank99 — APPROVED** [review]

**Review by rohan-kadkol — APPROVED** [review]
